src/vsr.zig

```diff
@@ -999,14 +1004,29 @@ pub fn quorums(replica_count: u8) struct {
     // The insight is that the replication phase is by far more common than the view change.
     // This trade-off allows us to optimize for the common case.
     // See the comments in `constants.zig` for further explanation.
+    assert(quorum_view_change >= 2 or quorum_view_change == replica_count);
     assert(quorum_view_change >= majority);
+    assert(quorum_view_change + quorum_replication > replica_count);
 
     return .{
         .replication = quorum_replication,
         .view_change = quorum_view_change,
     };
 }
 
+test "quorums" {
+    if (constants.quorum_replication_max != 3) return;
```

Comment:
```suggestion
    if (constants.quorum_replication_max != 3) return error.SkipZigTest;
```

?

Though, I've personally never got the benefit of marking the tests as skipped vs just sucesfull, but, if its easy to do in Zig, then why not?

Reply:
Thanks! I did not know this was a thing :smile: